File: calcPerformance.cpp
#include <stdint.h>
#include <unistd.h>
#include <stdio.h>
#include <stdlib.h>
#include <getopt.h>
#include <fcntl.h>
#include <sys/ioctl.h>
#include <linux/types.h>
#include <linux/spi/spidev.h>
#include <limits.h>
#include "time.h"

/*
#include <opencv2/opencv.hpp>
#include <opencv2/imgcodecs/imgcodecs.hpp>
#include <opencv2/highgui/highgui.hpp>
#include <opencv2/imgproc/imgproc.hpp>

#include "my_lepton.hpp"


using namespace std;
using namespace cv;
*/

int main(void)
{
    int ret, i,j;
    int sum = 0;
    clock_t start,finish;
    double during;
    start =clock();
    for(i=0;i<10000;i++)
    {
        for(j=0;j<10000;j++)
        {
            sum = sum+i*i;
        }
    }
    finish = clock();
    during = (double)(finish-start)/CLOCKS_PER_SEC;
    printf("time diff:%f second\n",during);

    return 0;
}

File: CalcPerformance.py
from timeDiff import timeDiff
import time
#timeCheck = timeDiff()
def calc():
    sum =0
    time0 = time.time()
    for i in range(0,10000):
        for j in range(0,10000):
            sum =sum +i*i
    time1= time.time()
    diff = time1-time0
    print("the time diff is :%f" %diff)

    
if __name__=='__main__':
    st = calc()
    
